src/_posts/compliance/2000-01-01-hds.md

```diff
@@ -70,3 +76,9 @@ La certification HDS de Scalingo couvre les périmètres suivants :
 Veuillez toujours vous référer à la page actuelle pour obtenir les dernières informations sur notre conformité HDS
 
 [Télécharger le certificat HDS 1.1 en vigueur](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
+
+## 🇫🇷 Qu'est-ce qu'un "Point de contact professionnel de santé" ? {#fr-hds-point-of-contact-health-professional}
+
+La certification HDS impose à Scalingo de maintenir une liste de *points de contact professionnel de santé* pour chacun de ses clients.
+Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer des relations avec le patient, etc.)
```

Comment:
```suggestion
Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer un incident de sécurité, etc.)
```

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2